packages/react-html/src/tests/HTML.test.tsx

```diff
@@ -140,6 +140,22 @@ describe('<HTML />', () => {
       expect(html.find('head').contains(link)).toBe(true);
     });
 
+    it('includes the script component', () => {
+      const script = (
+        <script
+          type="text/javascript"
+          dangerouslySetInnerHTML={{__html: 'alert("hi")'}}
```

Comment:
Would it also make sense to test this API?


```jsx
<Helmet script={[{innerHtml: 'some js'}]} />
```

Reply:
I think that might be more testing how Helmet works than testing our api. It might be a good idea to change the tests to just check that the right Helmet render methods were called though.

Reply:
Similarly, we might want to document that doing regular helmet stuff will work with this out of the box.